go/vt/vitessdriver/fakeserver_test.go

```diff
@@ -172,10 +178,11 @@ func CreateFakeServer() vtgateservice.VTGateService {
 }
 
 var execMap = map[string]struct {
```

Comment:
we should take the opportunity of refactoring and cleaning up this global variable. this map could live inside the `fakeVTGateService` instead maybe?